classes/01class/exercises/c01-e03/luizfds/c01-e03.txt

```diff
@@ -0,0 +1,36 @@
+To achieve exercise 03 from class 01 I've accessed AWS Console using a non-root user and went to EC2 option to create 2 new instances inside the default VPC. For the first one, I've opted to create an Amazon Linux AMI 2018.03.0 using the Free tier only option. In the 'Configure Instance' tab I've enabled the option to 'Auto-assign Public IP' for this first EC2 Instance (I left all other options as default), then I configured a Security Group named 'Luiz SG' with SSH protocol to open to network traffic on port 22. Before finish and create my instance I've named a Key pairs as 'Exercise: C01-E03' and downloaded it. The second instance was created almost in the same way but this time I've disabled the option 'Auto-assign Public IP' and using the same Security Group and key pair downloaded before.
+
+
+To be able to connect to the first instance I've used PuTTY and on the Terminal screen from it I got those values:
+
+Using username "ec2-user".
+Authenticating with public key "imported-openssh-key"
+
+       __|  __|_  )
+       _|  (     /   Amazon Linux AMI
+      ___|\___|___|
+
+https://aws.amazon.com/amazon-linux-ami/2018.03-release-notes/
+2 package(s) needed for security, out of 5 available
+Run "sudo yum update" to apply all updates.
+[ec2-user@ip-172-31-16-166 ~]$
+
+
+I couldn't connect to the second instance straight way because it only have a private IP, so I need to allocate and associate an Elastic IP for it using AWS Console Elastic IPs option. After done that I could connect to it and I got those values on Terminal screen:
```

Comment:
Hey Luiz, thats good to see that you're have put some time into that...
In that case, the second instance can not not have any external IP, so an Elastic IP attached to it is not the way to go.
The way to connect to the second instance is through a jump server or bastion host. This means that you need to access one server that have a public ip(like the first server) and from that public accessible server, you connect to the second server that only have internal IP. That's the reason for the name jump server. You first need to get into the first server to "jump" into the second server.
Review that and update your c01-e03.txt file so we can approve, but great job!

Reply:
Hi Kiko, thanks for the feedback.
I have done that change requested, please have a look and let me know if that's okay now.
Cheers,